### DISCUSSION TITLE
Sumatra doesn't understand folder and file names with spaces in the file path

### DISCUSSION THREAD

**Discussion by VladimirPVV** [sumatrapdfreader/sumatrapdf#2876] [General]
![Без имени](https://user-images.githubusercontent.com/111414516/185048694-d38de9f8-05d0-4edc-b32d-8cde68e5e19d.png)

**Comment by GitHubRulesOK** [comment]
drag and drop should not be a problem
![image](https://user-images.githubusercontent.com/12754913/185130404-073df5da-00cd-48e2-9c76-091f631bb24d.png)
![image](https://user-images.githubusercontent.com/12754913/185131291-9afcf49b-d2be-44cf-ac1c-9eb6079a2a6b.png)
